cms/src/test/java/uk/nhs/digital/externalstorage/resource/TestS3NodeMetadata.java

```diff
@@ -0,0 +1,48 @@
+package uk.nhs.digital.externalstorage.resource;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.apache.sling.testing.mock.jcr.MockJcr;
+import org.junit.Test;
+import uk.nhs.digital.externalstorage.ExternalStorageConstants;
+
+import javax.jcr.Node;
+import javax.jcr.Repository;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+public class TestS3NodeMetadata {
```

Comment:
How about calling it `St3NodeMetadataTest` so that the name of the class doesn't sound like a command but is closeer to a noun, instead.